sdk/sql/armsql/autorest.md

```diff
@@ -0,0 +1,12 @@
+### AutoRest Configuration
+
+> see https://aka.ms/autorest
+
+``` yaml
+require:
+- https://github.com/Azure/azure-rest-api-specs/blob/56cd4784756e65c303ed7aba007ffa8c7540a587/specification/sql/resource-manager/readme.md
+- https://github.com/Azure/azure-rest-api-specs/blob/56cd4784756e65c303ed7aba007ffa8c7540a587/specification/sql/resource-manager/readme.go.md
+module-version: 0.1.0
+modelerfour:
+  lenient-model-deduplication: true
```

Comment:
We cannot merge this until we get explicit explain of why we need to deduplicate this.
I will follow up on this.